autotest.pm

```diff
@@ -345,9 +345,11 @@ sub runalltests {
         if (!$vmloaded && $fullname eq $firsttest) {
             if ($bmwqemu::vars{SKIPTO}) {
                 if ($bmwqemu::vars{TESTDEBUG}) {
+                    $current_test->record_resultfile('loading snapshot', 'TESTDEBUG set');
```

Comment:
are you sure this works without adding `result => 'ok'`?

Reply:
Good question. And "works" means there's no `undef` being recorded (but a meaningful result) - even if it has no visual impact.

Reply:
It's documented to be allowed, and results in `unk` afair

Reply:
Well, but as here there is nothing failing I recommend to mark the step as "ok", or why do you want to differ from the original proposal?